### ISSUE TITLE
latest eclipse 2024-03 is completely hosted, java.lang.NoClassDefFoundError: org/eclipse/swt/widgets/Display

### ISSUE THREAD

**Issue by crowlogic** [eclipse-aspectj/ajdt#52]
No JDT editor appears, just blank workspaces and exceptions

this is actually a disgrace, i havent had an upgrade break like this since using eclipse sine 2007

**Comment by kriegaex** [comment]
Hosted? What do you mean. And what is a disgrace? Your error description, maybe. Closing until you provide further details, then we can reopen. Thanks in advance.

**Comment by crowlogic** [comment]
i just had to reinstall from scratch, try upgrading instead of fresh install.. my descriptions disgraceful apperance was in direct propotion to the disgrace of the failure to upgrade from one release to the next without a working fallback method. have a fine day i reinstalled from scratch and am not sufficiently  motivated to revisit the problem. thank  you

**Comment by crowlogic** [comment]
Hosed. Typo . Kaput . Both of my machines running the 2023 - 12 release.
Add this problem when upgrading to 2020 four, - 3.

**Comment by kriegaex** [comment]
With all due respect: Did you just open this issue here to rant, or do want to help yourself and others? You complained about exceptions. Where are they? Why did you not include or attach them to the issue? Like this, I cannot even check if the problem was really caused by AJDT or other components. Are you really expecting me to be able with the information given in your issue description?

If your problem was like mine and other people's in #49, that was fixed yesterday. I always was able to click the error box away and continue working in non-AspectJ projects or start updates or go to menu entry "reinstall new software", selecting the 2024-03-specific update site and fix the problem. I tried several times in the process of adjusting AJDT to the API changes. If you experienced something more severe than that annoying error box, it would be important to reproduce it and find out why an update was not working for you.

Is your problem now solved after the reinstall? Is AJST working? You did not say clearly.

> am not sufficiently motivated to revisit the problem

Yeah, that's helpful to everyone. At least you were sufficiently motivated to rant and make claims without any proofs, wasting your own time and mine.
